x-pack/auditbeat/module/system/packages/_meta/docs.asciidoc

```diff
@@ -0,0 +1,8 @@
+The System `packages` metricset provides ... TODO.
```

Comment:
Can you include the information of what packaging systems are currently supported? I think dpkg/deb and Homebrew, right? And probably the next line with the OS support is superfluous.